ui/app/appointments/controllers/manage/appointmentsSummaryController.js

```diff
@@ -54,10 +58,12 @@ angular.module('bahmni.appointments')
                             all: appointmentCount.allAppointmentsCount + total.all,
                             missed: appointmentCount.missedAppointmentsCount + total.missed
                         };
-                    }, {all: 0, missed: 0});
+                    }, { all: 0, missed: 0 });
                     $scope.weekDatesInfo.push(weekDate);
+                    console.log(weekDate);
                 }
+                console.log($scope.weekDatesInfo);
             };
 
-            return init();
+            // return init();
```

Comment:
If this is not needed anymore lets rather remove instead of commenting out. Too much commented code can leave the code base a bit messy.